Documentation/Books/Manual/Architecture/DeploymentModes/Cluster/Architecture.md

```diff
@@ -144,17 +172,41 @@ is synchronous.
 Synchronous replication works on a per-shard basis. Using the option _replicationFactor_,
 one configures for each _collection_ how many copies of each _shard_ are kept in the Cluster.
 
+{% hint 'danger' %}
+If a collection have a _replication factor_ equal to 1, its data is **not**
+replicated in other _DBServers_. This exposes you to a risk of data loss, if
+the machine where the _DBServer_ that is storing this collection permanently
+fails.
+{% endhint %}
+
+{% hint 'warning' %} 
+The _replication factor_ has to be set to a value equals or higher than 2
+to achieve minimal data redundancy via the synchronous replication.
```

Comment:
Shouldn't this be part of the previous hint?

Reply:
done - the previous one is a danger one though.